packages/map-popup/src/util.ts

```diff
@@ -65,3 +65,32 @@ export function makeDefaultGetEntityName(
     return stationName;
   };
 }
+
+/**
+ * Helper method to find the element within the app menu at the given offset
+ * (e.g. previous or next) relative to the specified element.
+ *
+ * @param {string} query  - Argument that gets passed to document.querySelectorAll
+ * @param {HTMLElement} element - Specified element (e.target)
+ * @param {1 | -1} offset - Determines direction to move within array of focusable elements (previous or next)
+ * @returns {HTMLElement} - element to be focused
+ */
+
+export function getEntryRelativeTo(
```

Comment:
It would make sense to:
1. Find a better name... How about `getSibling` or `getSiblingElement` or `getRelativeElement`?
2. Create two exported wrapper methods: `focusNextSibling`/`focusNextElement(query, element)` and `focusPreviousSibling`/`focusPreviousElement(query, element)` that would replace `getEntryRelativeTo(queryId, element, +1/-1)?.focus()`. That's the code we ultimately use in a few other places in OTP-RR.